packages/block-library/src/navigation-menu-item/editor.scss

```diff
@@ -28,10 +28,11 @@ $menu-label-field-width: 140px;
 	}
 }
 
-
 .wp-block-navigation-menu-item {
+	background-color: var(--background-color-menu-link);
+	color: var(--color-menu-link);
```

Comment:
what do you think @marekhrabe @shaunandrews about supporting IE11?

Reply:
Gutenberg supports IE11 so we must follow I guess https://make.wordpress.org/core/handbook/best-practices/browser-support/

I'm not sure how to do it here. We can pass the text color through `inherit`/`currentColor` but we cannot do the same for background :/

Reply:
Passing the background color to each navigation-menu item could be the logical solution, the question is how since it renders them using the `<InnerBlocks />` component and I don't know how to propagate and react a property to the built components.